api/resource/resource.go

```diff
@@ -37,6 +39,8 @@ var BuildAnnotations = []string{
 	utils.BuildAnnotationAllowKindChange,
 }
 
+const OriginAnnotation = konfig.ConfigAnnoDomain + "/origin"
```

Comment:
can we keep all these constants annotation names in one place?
Either here or in `makeResIds.go`?

Reply:
added a new file. utils/annotations.go, for these constants to go.